tests/tensors/test_ckks_vector.py

```diff
@@ -305,13 +329,19 @@ def test_mul_inplace(context, vec1, vec2):
         ([1, 2, 3, 4], [4, 3, 2, 1]),
         ([-1, -2], [-73, -10]),
         ([1, 2], [-73, -10]),
+        ([1, 2, 3, 4], 2),
+        # ([1, 2, 3, 4], 0),
```

Comment:
The result is a vector of zeros, which is a transparent ciphertext.
Since `SEAL_THROW_ON_TRANSPARENT_CIPHERTEXT` is enabled SEAL will throw an error.

Reply:
There is an open issue #36 about that.

Reply:
Maybe we should enable the testcase and explicitly check for exceptions? so that everybody is aware of the limitation?

I think it's better than leaving it commented, that's all.

Reply:
Because I think this exception might not be good to be forwarded to the user, this should be dealt with internally, and we didn't fix issue #36 yet, so left it there to not forget about it.